maxim/tests/test_maxim_core_simple.py

```
@@ -0,0 +1,246 @@
+
+    def tearDown(self):
+        """Clean up after tests."""
+        if hasattr(Maxim, "_instance"):
+            delattr(Maxim, "_instance")
+        if "MAXIM_API_KEY" in os.environ:
+            del os.environ["MAXIM_API_KEY"]
+
+    def test_enable_prompt_management_with_none_cache(self):
+        """Test enable_prompt_management with cache=None."""
+        maxim = Maxim({"api_key": "test-key", "prompt_management": False})
+        original_cache = getattr(maxim, "_Maxim__cache")
```

Ruff (0.11.9)

Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)